src/Graphics/src/Graphics/PathF.cs

```diff
@@ -1452,6 +1452,283 @@ public RectF GetBoundsByFlattening(float flatness = 0.001f)
 			_cachedBounds = new RectF(l, t, r - l, b - t);
 			return (RectF)_cachedBounds;
 		}
+	public RectF CalculateTightBounds()
```

Comment:
This is a change that will require update in the publicAPIs. Because this class is available in all the platforms, must update the `PublicAPI.Unshipped.txt` from Graphics in all the platforms.